### PR TITLE
feat: add table level alias for select, insert and update

### LINKED ISSUE #264: add Table.select?
Has allowing `Table` to `select`, eliminating the name to re-specify a table name with `from_` been considered? e.g.

```python
from pypika import Table, Query
t = Table('my_table')
t.select(t.x, t.y) == Query.from_('my_table').select(t.x, t.y) # -> True
```

Just an idea. Seems like it would add convenience when using from console/jupyter.

Thanks for your hard work on a great library!

### PR DESCRIPTION
**Pull request by harshanarayana**
Fixes #264 and #259 

### To Do

- [x] Add documentation about table level aliases
- [x] Add example in documentation

### CODE REVIEW

**Comment by harshanarayana** [comment]
Tests are failing due to an issue in CI pipeline. Fix present in PR #266

File: pypika/queries.py [hunk 1]
```diff
@@ -148,6 +148,15 @@ def __ne__(self, other):
     def __hash__(self):
         return hash(str(self))
 
+    def select(self, *terms):
+        return Query.from_(self._table_name).select(*terms)
```
Comment (twheys) [inline pypika/queries.py]: Need to pass in the instance of Table to `from_` in order to carry all of the information over. Can be just changed to `Query.from_(self) ... `

Comment (harshanarayana) [reply pypika/queries.py]: @twheys Sure. That makes sense. Let me change that

**Comment by twheys** [comment]
Are you planning on completing the TODOs? I was waiting on those before merging

**Comment by harshanarayana** [comment]
@twheys Yes. I will update the PR with the missing items by EOD today.

**Comment by harshanarayana** [comment]
@twheys 🔔 I've updated the PR a couple of days ago to include all the todo items

**Comment by twheys** [comment]
sorry for the delay. Merged and published.

**Comment by harshanarayana** [comment]
@twheys No worries. I forgot to add a message when I pushed the changes and totally forgot about it

**Pull request merged**